src/main/java/org/homeschoolpebt/app/utils/SubmissionUtilities.java

```diff
@@ -187,7 +184,69 @@ public static String getStandardOperatingExpensesAmount(Map<String, Object> fiel
       return "";
     }
     Float grossAmount = Float.parseFloat(rawGrossAmount.toString());
-    return decimalFormat.format(grossAmount * 0.4);
+    return decimalFormatWithoutComma.format(grossAmount * 0.4);
+  }
+
+  public static String getSelfEmployedOperatingExpensesAmount(Map<String, Object> fieldData) {
+    boolean useCustomOperatingExpenses = (
+        fieldData.get("incomeSelfEmployedCustomOperatingExpenses") != null &&
+            fieldData.get("incomeSelfEmployedCustomOperatingExpenses").equals("true") &&
+            fieldData.get("incomeSelfEmployedOperatingExpenses") != null
+    );
+    double expenses;
+    if (useCustomOperatingExpenses) {
+      expenses = Double.parseDouble(fieldData.get("incomeSelfEmployedOperatingExpenses").toString());
+    } else {
+      Object rawGrossAmount = fieldData.get("incomeGrossMonthlyIndividual");
+      if (rawGrossAmount == null) {
+        // TODO: Null handling? Redirect?
+        expenses = 0;
+      } else {
+        double grossMonthly = Double.parseDouble(rawGrossAmount.toString());
+        expenses = 0.4 * grossMonthly;
+      }
+    }
+    return formatMoney(String.valueOf(expenses));
+  }
+
+  public static String getSelfEmployedNetIncomeAmount(Map<String, Object> fieldData) {
+    Object rawGrossAmount = fieldData.get("incomeGrossMonthlyIndividual");
+    if (rawGrossAmount == null) {
+      // TODO: Null handling? Redirect?
+      return "";
+    }
+    double grossMonthly = Double.parseDouble(rawGrossAmount.toString());
+    boolean useCustomOperatingExpenses = (
+        fieldData.get("incomeSelfEmployedCustomOperatingExpenses") != null &&
+            fieldData.get("incomeSelfEmployedCustomOperatingExpenses").equals("true") &&
+            fieldData.get("incomeSelfEmployedOperatingExpenses") != null
+    );
+
+    double netMonthly;
+    if (useCustomOperatingExpenses) {
+      netMonthly = grossMonthly - Double.parseDouble(fieldData.get("incomeSelfEmployedOperatingExpenses").toString());
+    } else {
+      netMonthly = 0.6 * grossMonthly;
+    }
+    return formatMoney(String.valueOf(netMonthly * 12));
```

Comment:
We know this file is a little bit of a mess, but since we also know we're going to keep revising the income flow, we are OK cleaning this up later.